friktion/friktion/swap.py

```diff
@@ -35,6 +35,8 @@
 from .swap_order_template import SwapOrderTemplate
 
 
+GLOBAL_FRIKTION_AUTHORITY = PublicKey("7wYqGsQmfVigMSratssoPddfLU1P5srZcM32nvKAgWkj")
```

Comment:
will this be the same for all vaults and for both devnet and mainnet?

Reply:
yes (based on the meet discussion)

Reply:
Why not in https://github.com/Friktion-Labs/paradigm-sdks/blob/1eb4ded1920a20c0f3ccdf789369f6d57cc70334/friktion/friktion/constants.py ?